PF/Assignment4/NQueenTest.java

```diff
@@ -0,0 +1,59 @@
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class NQueenTest {
+	int[][] expected = new int[][] { { 0, 1, 0, 0 }, { 0, 0, 0, 1 },
```

Comment:
Since these are not common for all test method, they should not be class members rather they should be local to each test methods. Make it sure next time :)